src/encoded/static/components/publication.js

```diff
@@ -7,7 +7,54 @@ import { DbxrefList } from './dbxref';
 import { PickerActions } from './search';
 import Status from './status';
 import { DisplayAsJson } from './objectutils';
-
+import { SortTablePanel, SortTable } from './sorttable';
+import { FetchedData, Param } from './fetched';
+
+
+const datasetsColumns = {
+    accession: {
+        display: dataset => <a href={dataset['@id']}>{dataset.accession}</a>,
+        title: 'Accession',
+    },
+    type: {
+        getValue: datasets => (datasets['@type'] || [{}])[0], // only the first matters
+        title: 'Type',
+    },
+    target: {
+        getValue: dataset => (dataset.target || {}).label,
+        title: 'Target',
+    },
+    assay_term_name: {
+        getValue: (dataset) => {
+            // assay_term_name can be a string or an array
+            if (typeof (dataset.assay_term_name) === 'string') {
+                return dataset.assay_term_name;
+            }
+            return (dataset.assay_term_name || []).join(', ');
+        },
+        title: 'Assay',
+    },
+    description: {
+        title: 'Description',
+    },
+    lab: {
+        getValue: dataset => (dataset.lab || {}).title,
+        title: 'Lab',
+    },
+    biosample_summary: {
+        getValue: (dataset) => {
+            if (typeof (dataset.biosample_summary) === 'string') {
+                return dataset.biosample_summary;
+            }
+            return (dataset.biosample_summary || []).map(b => b.term_name).join(', ');
```

Comment:
Just wanted to confirm `experiment.biosample_summary` can be an array of objects? I skimmed over experiment.py and don’t see where that happens, and all other uses of `experiment.biosample_summary` in front-end code assumes it’s a string. If it can be an array, it seems like those have to be changed too.